src/components/MerchantsPage/gam/GiftAMealPage.tsx

```diff
@@ -80,6 +80,9 @@ const GiftAMealPage = (props: Props) => {
       )}
 
       <div className={styles.videoContainer}>
+        <VideoComponent videoId="FYlUOhzYlRM"></VideoComponent>
+        <h5 className={styles.videoDescription}>
+          Watch how we bring impact to our communties through Gift-a-Meal.
```

Comment:
This is causing a build error and was incorrectly included in the resolving the merge conflicts